pytest_rerunfailures.py

```diff
@@ -59,12 +68,10 @@ def check_options(config):
         config.pluginmanager.register(config._resultlog)
 
 
-def pytest_runtest_protocol(item, nextitem):
-    """
-    Note: when teardown fails, two reports are generated for the case, one for
-    the test case and the other for the teardown error.
-    """
+def reruns_count(item):
```

Comment:
I generally prefer to have function names which start with a verb, e.g. `get_maximum_reruns` to enhance readability of the code.